definitions/npm/@material-ui/core_v1.x.x/flow_v0.58.x-/core_v1.x.x.js

```diff
@@ -1594,7 +1741,7 @@ declare module "material-ui/styles/colorManipulator" {
   };
```

Comment:
Been around for a while but there's a typo on the line above (can't seem to add a comment in there...)

'ligthen' -> 'lighten'.

Thanks for the great definitions